churchland_pipeline/tasks/pacman.py

```diff
@@ -0,0 +1,216 @@
+        ---
+        force_polarity : tinyint # indicates whether pushing (polarity = 1) or pulling (polarity = -1) moves Pac-Man upwards
+        force_max : tinyint unsigned # maximum force [Newtons]
+        force_offset : decimal(5,4) # force offset to compensate for arm weight [Newtons]
+        target_type : char(3) # type code
+        target_offset : decimal(5,4) # offset
+        target_amplitude : decimal(5,4) # amplitude
+        target_duration : decimal(5,4) # duration
+        target_frequency1 : decimal(5,4) # primary frequency (initial, for chirp forces)
+        target_frequency2 : decimal(5,4) # secondary frequency (final, for chirp forces)
+        target_power : decimal(5,4) # power exponent
+        target_pad : decimal(5,4) # pad duration
+        stim_current : smallint unsigned # stim current (uA)
+        stim_electrode : smallint unsigned # stim electrode number
+        stim_polarity : tinyint unsigned # stim polarity
+        stim_pulses : tinyint unsigned # number of pulses in stim train
+        stim_width1 : smallint unsigned # first pulse duration (us)
+        stim_width2 : smallint unsigned # second pulse duration (us)
+        stim_interphase : smallint unsigned # interphase duration (us)
+        stim_frequency : smallint unsigned # stim frequency (Hz)
+        """
+
+    class Trial(dj.Part):
+        definition = """
+        # Trial data
+        -> master
+        trial_number : smallint unsigned # trial number (within session)
+        ---
+        -> Task.Condition
+        save_tag : tinyint unsigned # save tag
+        valid_trial : tinyint unsigned # is valid trial (1=yes, 0=no)
+        successful_trial : tinyint unsigned # is successful trial (1=yes, 0=no)
+        simulation_time : longblob # absolute simulation time
+        task_state : longblob # task state IDs
+        force_raw_online : longblob # amplified output of load cell
+        force_filt_online : longblob # online (boxcar) filtered and normalized force used to control Pac-Man
+        stim : longblob # ICMS delivery
+        reward : longblob # reward delivery
+        photobox : longblob # photobox signal
+        """
+    
+@schema
+class TrialAlignment(dj.Imported):
+    definition = """
+    -> acquisition.SyncChannel
+    trial_number : smallint unsigned # trial number (within session)
+    ---
+    alignment_index : int unsigned # alignment index (in Speedgoat time base)
+    speedgoat_alignment : longblob # alignment indices for Speedgoat data
+    ephys_alignment : longblob # alignment indices for Ephys data
+    """    
+
+# -------------------------------------------------------------------------------------------------------------------------------
+# LEVEL 1
+# -------------------------------------------------------------------------------------------------------------------------------
+
+@schema
+class Force(dj.Computed):
+    definition = """
+    # Single trial force
+    -> Task.Trial
+    -> TrialAlignment
```

Comment:
Are there multiple alignments per trial? Should `TrialAlignment` be moved out of the primary key?